tensorflow_recommenders_addons/dynamic_embedding/core/BUILD

```diff
@@ -76,8 +76,6 @@ custom_op_library(
         "kernels/segment_reduction_ops.h",
         "kernels/segment_reduction_ops_impl.cc",
         "kernels/segment_reduction_ops_impl.h",
-        "kernels/sparse_fill_empty_rows_op.cc",
```

Comment:
Hi @Lifann, please help confirm the deletion would not effect your system. Thanks!

Reply:
The SparseFillEmptyRows op was already supported in tf. And it makes no effects for me?